combat-trainer.lic

```diff
@@ -1298,7 +1301,9 @@ class SpellProcess
   def check_offensive(game_state)
     return if game_state.casting
     return if game_state.npcs.empty?
+    return if @offensive_spells_max_threshold && game_state.npcs.length > @offensive_spells_max_threshold
```

Comment:
game_state.npcs.length should be magic mindstates

Reply:
I completely misread the other threshold, wtf.